docs/other/install-cvmfs-stratum1.md

```diff
@@ -133,14 +133,19 @@ RewriteRule ^/cvmfs/(.*)$ /srv/cvmfs/$1
   ExpiresByType application/json "access plus 61 seconds" 
 </Directory>
 
-WSGIDaemonProcess cvmfs-api processes=2 display-name=%{GROUP} \
+WSGIDaemonProcess cvmfs-api threads=64 display-name=%{GROUP} \
     python-path=/usr/share/cvmfs-server/webapi
-WSGIProcessGroup cvmfs-api 
+<Directory /var/www/wsgi-scripts/cvmfs-server>
+  WSGIProcessGroup cvmfs-api
+  WSGIApplicationGroup cvmfs-api
+  Options ExecCGI
+  SetHandler wsgi-script
+  Require all granted
+</Directory>
 WSGISocketPrefix /var/run/wsgi 
-WSGIScriptAliasMatch /cvmfs/([^/]+)/api /var/www/wsgi-scripts/cvmfs-api.wsgi/$1 
 ```
 
-On EL6-based systems (apache httpd 2.2) replace the "Require all granted" with "Order allow, deny" and "Allow from all".
+On EL6-based systems (apache httpd 2.2) replace the "Require all granted" lines with "Order allow, deny" and "Allow from all".
```

Comment:
Maybe this isn't obvious to me because I don't have a config file in front of me but how does an admin decide when to replace `Required all granted` with `Order allow, deny` and when to replace it with `Allow from all`?

Reply:
They need to replace the one line (which now appears twice) with both the new lines, that's why it says "and".

Reply:
In that case, I think we need to be more explicit. Maybe something like this

> On EL6-based systems (apache httpd 2.2) replace `Require all granted`:
> * Under the `<Directory "/srv/cvmfs">` stanza, replace it with `Order allow, deny`
> * Under the `<Directory /var/www/wsgi-scripts/cvmfs-server>` stanza, replace it with `Allow from all`